patches/tools-grit-preprocess_if_expr.gni.patch

```diff
@@ -0,0 +1,34 @@
+diff --git a/tools/grit/preprocess_if_expr.gni b/tools/grit/preprocess_if_expr.gni
+index 850bff9aa94b4..2836f5f777627 100644
+--- a/tools/grit/preprocess_if_expr.gni
++++ b/tools/grit/preprocess_if_expr.gni
+@@ -55,6 +55,29 @@ template("preprocess_if_expr") {
+              "--in-files",
+            ] + invoker.in_files + grit_args
+ 
++    if (defined(invoker.override_files)) {
++      override_in_folder =
++          string_replace(get_path_info(rebase_path(in_folder), "abspath"),
++                         "//",
++                         "//brave/")
++
++      foreach(override, invoker.override_files) {
++        # Add dependency on brave override
++        inputs += [ "$override_in_folder/" + override ]
++      }
++
++      # We generate the override file outputs at a higher level so we don't need to do it multiple times
++      foreach(output_file, invoker.override_file_outputs) {
++        outputs += ["$out_folder/$output_file"]
++      }
++
++      args += [
++                "--override-in-folder",
++                rebase_path(override_in_folder, root_build_dir),
++                "--overrides",
++              ] + invoker.override_files
++    }
++
```

Comment:
Probably simpler to just have a one-line patch here and do the work in the override.

Reply:
Is there a way to add Chromium src overrides for gni files?

Reply:
I think this is the best you can do with .gni files https://github.com/brave/brave-browser/wiki/Patching-Chromium#patching-gngni-files